manager/api/export-import/src/main/java/io/apiman/manager/api/exportimport/manager/StorageImportDispatcher.java

```diff
@@ -68,8 +79,8 @@
 
     @Inject
     private IStorage storage;
-    @Inject @ApimanLogger(StorageImportDispatcher.class)
-    private IApimanLogger logger;
+    private IApimanLogger logger = ApimanLoggerFactory.getLogger(StorageImportDispatcher.class);
```

Comment:
Any reason to not make this also static final?

Reply:
See previous